class.jetpack.php

```diff
@@ -5576,7 +5590,13 @@ public static function jumpstart_has_updated_module_option( $option_name = '' )
 
 		if ( in_array( $option_name, $option_names ) ) {
 			Jetpack_Options::update_option( 'jumpstart', 'jetpack_action_taken' );
+
+			//Jump start is being dismissed send data to MC Stats
+			Jetpack::init()->stat( 'jumpstart', 'manual,'.$option_name );
```

Comment:
In this line (and the ones like it) as you already have Jetpack in the local $jetpack variable, you don't need to reinit.  You can just call it as `$jetpack->stat()`